cmd/clusterctl/phases/createbootstrapcluster.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 func CreateBootstrapCluster(provisioner bootstrap.ClusterProvisioner, cleanupBootstrapCluster bool, clientFactory clusterclient.Factory) (clusterclient.Client, func(), error) {
-	klog.Info("Creating bootstrap cluster")
+	klog.Info("Preparing bootstrap cluster")
```

Comment:
I was a bit confused, this log is in the function of `CreateBootstrapCluster`, I think using `Creating` should be OK?

For your case, are you running `clusterctl` without creating bootstrap cluster? If so, can we avoid calling this function when using existing cluster?

Reply:
I think the fix is for when using the `existing` bootstrap cluster type, which it makes sense. It's a small change that works for any bootstrapper type.

Reply:
yes, it's a nit and not important ,just better to have 
since it take me some time to know it's not 'creating' at all, it's just preparing (either creating or reusing)

Reply:
this make sense, thanks @jichenjc @vincepri